scripts/merge_tables.R

```diff
@@ -113,22 +190,35 @@ merge_tables <- function(cwd, prefix){
 ###      MAIN      ###
 ######################
 # Write log
-if ( verb ) cat("Creating output directory...\n", sep="")
+if (verb)
+  cat("Creating output directory...\n", sep = "")
 
 # Create output directories
-dir.create(dirname(out.file), recursive=TRUE, showWarnings=FALSE)
+dir.create(dirname(out.file),
+           recursive = TRUE,
+           showWarnings = FALSE)
```

Comment:
Not a big fan of this line wrapping style. But more importantly, it's inconsistent with the one you use later, on lines 213-220 (which I prefer).

You can pick whichever one you prefer (regardless of my preference), but make sure it's being used consistently.